content/chainguard/chainguard-enforce/iam-groups/rolebinding-terraform-gh/index.md

```diff
@@ -0,0 +1,393 @@
+---
+title: "Create Rolebindings for a GitHub Team Using Terraform"
+linktitle: "GitHub Team Role Binding"
+description: "Procedural tutorial outlining how to use Terraform to create Chainguard rolebindings for members of a GitHub team."
+type: "article"
+date: 2023-06-10T08:48:45+00:00
+lastmod: 2023-06-10T08:48:45+00:00
+draft: false
+tags: ["Enforce", "Product", "Procedural"]
+images: []
+weight: 050
+---
+
+There may be cases where an organization will want multiple users to have access to the same Chainguard Enforce instance. Originally, Chainguard allowed you to grant other users access to Enforce by [generating an invite link or code](/chainguard/chainguard-enforce/iam-groups/how-to-manage-iam-groups-in-chainguard-enforce/#inviting-others-to-a-group).
```

Comment:
What do you think of reframing this a bit? I am not sure if we need to use the time signals since users may not be aware that one feature was released by another. I also think it may be better to highlight that using Terraform is an addition rather than the other method could be negative.... for example:

> "Granting access through Terraform ensures that you are not risking unwanted users having access to Enforce..." 

So this is additive vs. showing how the manual way could "risk unwanted users".

Reply:
That's a lot better, thank you Lisa